lib/puppet_x/certs/provider/keystore.rb

```diff
@@ -25,6 +42,8 @@ def type
         def generate_keystore
           temp_alias = 'temporary-entry'
 
+          FileUtils.rm_f(store)
```

Comment:
Does `rm_f` handle file not found?

Reply:
Yes. https://ruby-doc.org/stdlib-2.4.1/libdoc/fileutils/rdoc/FileUtils.html#method-i-safe_unlink states it's equal to `FileUtils.rm(file, force: true)` which ignores those errors.